controllers/toolchainstatus/toolchainstatus_controller_test.go

```diff
@@ -90,12 +93,16 @@ func prepareReconcile(t *testing.T, requestName string, httpTestClient *fakeHTTP
 			}
 			return clusters
 		},
+		VersionCheckManager: status.VersionCheckManager{GetGithubClientFunc: mockedGetGitHubClient, LastGHCallsPerRepo: map[string]time.Time{
+			"host-operator":        lastGitHubAPICall,
+			"registration-service": lastGitHubAPICall,
+		}},
 	}
 	return r, reconcile.Request{NamespacedName: test.NamespacedName(test.HostOperatorNs, requestName)}, fakeClient
 }
 
 func prepareReconcileWithStatusConditions(t *testing.T, requestName string, memberClusters []string, conditions []toolchainv1alpha1.Condition, initObjs ...runtime.Object) (*Reconciler, reconcile.Request, *test.FakeClient) {
-	reconciler, req, fakeClient := prepareReconcile(t, requestName, newResponseGood(), memberClusters, initObjs...)
+	reconciler, req, fakeClient := prepareReconcile(t, requestName, newResponseGood(), mockLastGitHubAPICall, test.MockGitHubClientForRepositoryCommits(buildCommitSHA, time.Now().Add(-time.Hour*1)), memberClusters, initObjs...)
```

Comment:
correct me if I'm wrong, but it looks like that in most cases you use the same mocked Github client. to make the test code a bit easier to read, you could store it as a variable and reuse that
```go
var defaultGitHubClient = test.MockGitHubClientForRepositoryCommits(buildCommitSHA, time.Now().Add(-time.Hour*1))
```
```suggestion
	reconciler, req, fakeClient := prepareReconcile(t, requestName, newResponseGood(), mockLastGitHubAPICall, defaultGitHubClient, memberClusters, initObjs...)
```

Reply:
agree, this makes it shorter and easier to read, I've changed it in [afac3d4](https://github.com/codeready-toolchain/host-operator/pull/819/commits/afac3d4884ccae631340c95f990a4e7c0eec95bf) . Thanks!